.kokoro

```diff
@@ -55,6 +56,12 @@ popd
 
 fix_bazel_imports
 
-./.kokoro-ios-runner/bazel.sh test //:UnitTests 8.1.0
+if [ -n "$KOKORO_BUILD_NUMBER" ]; then
+  bazel version
+  use_bazel.sh "$BAZEL_VERSION"
+  bazel version
+fi
+
+./.kokoro-ios-runner/bazel.sh test //:UnitTests 9.0.0
```

Comment:
Tip: Xcode 10?

Reply:
Xcode 10 is not yet available on kokoro unfortunately.